### PR TITLE
[WIP] Windows installer improvements

### PR DESCRIPTION
**Pull request by sblaisot**
- [x] Improve warning messages dialogs
- [ ] Improve upgrade path : automatically launch older version uninstaller and explain user what to do

### CODE REVIEW

File: build/wix/Localization/mixxx_de-DE.wxl [hunk 1]
```diff
@@ -26,7 +26,7 @@
   <String Id="NewerInstalled">Die gleiche oder eine neuere Version dieses Produkts ist bereits installiert.</String>
   <String Id="x64VersionRequired">Sie müssen die 64-Bit-Version dieses Produkts auf 64-Bit-Windows installieren.</String>
   <String Id="x86VersionRequired">Sie müssen die 32-Bit-Version dieses Produkts auf 32-Bit-Windows installieren.</String>
-  <String Id="x86VersionWarning">Sie sind dabei, die 32-Bit-Version auf einem 64-Bit-Betriebssystem zu installieren.&#13;&#10;Wir empfehlen mit der 64-Bit-Version für bessere Performance.&#13;&#10;Sie können das der 64-Bit-Installationspaket von http://mixxx.org/download/ herunterladen.</String>
+  <String Id="x86VersionWarning">Sie sind dabei, die 32-Bit-Version auf einem 64-Bit-Betriebssystem zu installieren. Wir empfehlen mit der 64-Bit-Version für bessere Performance. Sie können das der 64-Bit-Installationspaket von http://mixxx.org/download/ herunterladen.</String>
```
Comment (daschuer) [inline build/wix/Localization/mixxx_de-DE.wxl]: Sie installieren gerade die 32-Bit-Version auf einem 64-Bit-Betriebssystem. Wir empfehlen stattdessen die 64-Bit-Version für eine bessere Performance zu installieren. Sie können ...

**Review by daschuer — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Are all translations auto generated? This It is probably a good idea to hold them back until a native speaker has prove read them.

**Comment by sblaisot** [comment]
Nope, these are translations from transifex. I only fond a way  to remove special chars from the text to make translation easier. Please update translations from transifex if needed.

**Comment by daschuer** [comment]
I cannot edit the transiflex translation. Do I need to get permission?

**Comment by sblaisot** [comment]
The transifex account daschuer2 is authorized to translate to the de and de_DE languages.
If you have any other request (other account or language), you can request it directly on the transifex website and I will be happy to add you to whatever language you need.

You will then be able to translate the windows installer strings to de_DE at this url: https://www.transifex.com/mixxx-dj-software/mixxxdj-windows-installer/translate/#de_DE/

**Comment by daschuer** [comment]
I can only add my text as a suggestion. 
I have just done it.

**Comment by daschuer** [comment]
So this PR can be merged now. Thank you.

**Comment by sblaisot** [comment]
oops, I didn't finished improving upgrade path ("WIP" in the subject, unchecked boxes in the description).
nevermind, I'll open a new PR for the next work ;)
Thanks for merging @daschuer

**Comment by Be-ing** [comment]
Any progress on automatically running the old uninstaller?

**Comment by Be-ing** [comment]
Great, thank you.

**Pull request merged**